includes/_header_meta.slim

```diff
@@ -18,6 +18,13 @@ meta name="viewport" content="width=device-width, initial-scale=1.0"
   meta name="twitter:url" content=@pod.or_cocoapods_url
   meta name="twitter:image:src" content=@pod.or_thumbnail_image
 
+  meta name="og:site_name" content="@CocoaPods"
```

Comment:
Good catch!

--  
Ash Furrow
https://ashfurrow.com/  

On May 4, 2016 at 2:58:27 PM, hugo tunius (notifications@github.com(mailto:notifications@github.com)) wrote:

>  
>  
> In includes/_header_meta.slim(https://github.com/CocoaPods/shared_resources/pull/18#discussion_r62095154):
>  
> > @@ -18,6 +18,13 @@ meta name="viewport" content="width=device-width, initial-scale=1.0" > meta name="twitter:url" content=@pod.or_cocoapods_url > meta name="twitter:image:src" content=@pod.or_thumbnail_image > > + meta name="og:site_name" content="@CocoaPods"  
>  
> Do we really want a Twitter style handle here?
>  
>  
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub(https://github.com/CocoaPods/shared_resources/pull/18/files/9c9bf6f79c56c70baa38dff78197382aa091131c#r62095154)
>  
>  
>  
>  
>